Jasmine/Tetris/ViewModels/TetrisTileView.swift

```diff
@@ -0,0 +1,17 @@
+import UIKit
+
+/// Represents the view for a tile
+class TetrisTileView: UILabel {
```

Comment:
Well you still need some discipline cos merge conflict will depend on physical file structure